docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,168 @@
+
+First Run
+=========
+
+Now you have installed MCDR, it's time to run it for the first time
+
+.. note::
+
+    Most common use of MCDR is to control a Minecraft server, so we also take this as the demonstrated use case here
+
+.. tip::
+
+    To manage Minecraft server with MCDR, It is recommended to have basic knowledge of common Minecraft server softwares
```

Comment:
Notes that "YAML and JSON files" are not part of common Minecraft server softwares, so I'll suggest to change this line to something like "have basic knowledge of Minecraft server management"